src/viewProviders/newResources.ts

```diff
@@ -620,6 +608,11 @@ export class NewResourceViewProvider
         ContextValues.directSchemaRegistryAvailable,
         directEnvs.some((env) => !!env.schemaRegistry),
       ),
+      setContextValue(
+        ContextValues.localKafkaClusterAvailable,
+        localEnv?.kafkaClusters.length !== 0,
```

Comment:
We'll need to change this for cases where `localEnv` is undefined, since it will pass the `!== 0` check:

<img width="1390" height="457" alt="Image" src="https://github.com/user-attachments/assets/f80c4d08-a233-4d76-b920-c0e5c0e552d8" />

```suggestion
        Array.isArray(localEnv?.kafkaClusters) && localEnv?.kafkaClusters.length !== 0,
```

Reply:
Current logic prevents local Kafka from starting since it thinks one is already available:
<img width="1046" height="834" alt="image" src="https://github.com/user-attachments/assets/eddb836c-bb5d-41ec-b589-48f5865f54c8" />